internal/servicedeployer/factory.go

```diff
@@ -67,28 +75,27 @@ func Factory(options FactoryOptions) (ServiceDeployer, error) {
 
 // FindDevDeployPath function returns a path reference to the "_dev/deploy" directory.
 func FindDevDeployPath(options FactoryOptions) (string, error) {
-	dataStreamDevDeployPath := filepath.Join(options.DataStreamRootPath, devDeployDir)
-	_, err := os.Stat(dataStreamDevDeployPath)
-	if err == nil {
+	dataStreamDevDeployPath := filepath.Join(options.DataStreamRootPath, options.DevDeployDir)
+	if _, err := os.Stat(dataStreamDevDeployPath); err == nil {
 		return dataStreamDevDeployPath, nil
 	} else if !errors.Is(err, os.ErrNotExist) {
 		return "", fmt.Errorf("stat failed for data stream (path: %s): %w", dataStreamDevDeployPath, err)
 	}
 
-	packageDevDeployPath := filepath.Join(options.PackageRootPath, devDeployDir)
-	_, err = os.Stat(packageDevDeployPath)
-	if err == nil {
+	packageDevDeployPath := filepath.Join(options.PackageRootPath, options.DevDeployDir)
+	if _, err := os.Stat(packageDevDeployPath); err == nil {
 		return packageDevDeployPath, nil
 	} else if !errors.Is(err, os.ErrNotExist) {
 		return "", fmt.Errorf("stat failed for package (path: %s): %w", packageDevDeployPath, err)
 	}
-	return "", fmt.Errorf("\"%s\" directory doesn't exist", devDeployDir)
```

Comment:
I think this was intentional, backslashes in Windows paths are escaped by `%q`, but are not by `%s`.